src/redis/Makefile

```diff
@@ -19,6 +19,7 @@ $(addprefix $(DEST)/, $(MAIN_TARGET)): $(DEST)/% :
 	dpkg-source -x redis_$(REDIS_VERSION_FULL).dsc
 
 	pushd redis-$(REDIS_VERSION)
+	export ARCH=""
```

Comment:
>export ARCH="" [](start = 1, length = 14)

why this is needed?

Reply:
deps/hiredis/Makefile 
REAL_CFLAGS=$(OPTIMIZATION) -fPIC $(CFLAGS) $(WARNINGS) $(DEBUG) $(ARCH) $(CPPFLAGS), will insert ARCH with without any option for ARM.